client/src/components/theme.ts

```diff
@@ -24,15 +34,31 @@ const spacing = {
   xxxs: 2,
 };
 
+export const spacesXxl = (props: CommonThemeProps) => getSpaces(props)?.xxl;
+export const spacesXl = (props: CommonThemeProps) => getSpaces(props)?.xl;
+export const spacesL = (props: CommonThemeProps) => getSpaces(props)?.l;
+export const spacesM = (props: CommonThemeProps) => getSpaces(props)?.m;
+export const spacesS = (props: CommonThemeProps) => getSpaces(props)?.s;
+export const spacesXs = (props: CommonThemeProps) => getSpaces(props)?.xs;
+export const spacesXxs = (props: CommonThemeProps) => getSpaces(props)?.xxs;
+export const spacesXxxs = (props: CommonThemeProps) => getSpaces(props)?.xxxs;
+export const spacesDefault = (props: CommonThemeProps) =>
+  getSpaces(props)?.default;
+
```

Comment:
thank you SO much for adding these!! Too bad SDS Hackday didn't add these in SDS library, so we don't have to duplicate the work in both Data Portal and Explorer. Hopefully it will be done in 2024!